src/Components/Server/ref/Microsoft.AspNetCore.Components.Server.netcoreapp3.0.cs

```diff
@@ -36,22 +34,6 @@ public sealed partial class CircuitOptions
         public System.TimeSpan DisconnectedCircuitRetentionPeriod { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } [System.Runtime.CompilerServices.CompilerGeneratedAttribute]set { } }
         public System.TimeSpan JSInteropDefaultCallTimeout { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } [System.Runtime.CompilerServices.CompilerGeneratedAttribute]set { } }
     }
-    public partial class ComponentPrerenderingContext
```

Comment:
There's no need for abstractions related to prerendering. It's just an MVC feature now.